### PR TITLE
cleanup Paramore.Brighter.Outbox.PostgreSql

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Review by iancooper — APPROVED** [review]

File: src/Paramore.Brighter.Outbox.PostgreSql/PostgreSqlOutboxSync.cs [hunk 1]
```diff
@@ -254,10 +238,8 @@ public void MarkDispatched(Guid id, DateTime? dispatchedAt = null, Dictionary<st
 
             try
             {
-                using (var command = InitMarkDispatchedCommand(connection, id, dispatchedAt))
-                {
-                    command.ExecuteNonQuery();
-                }
+                using var command = InitMarkDispatchedCommand(connection, id, dispatchedAt);
```
Comment (iancooper) [inline src/Paramore.Brighter.Outbox.PostgreSql/PostgreSqlOutboxSync.cs:241]: You can't use Using Declarations as we are C# 7.3 due to supporting .NET Standard 2.0 if you look in the file section of GitHub you should be able to see the build errors against the files.

(We support netstandard20 as this gives us framework and core support. Dropping framework support is something we might consider in V10, but the principle of a library being as compatible as important remains important to us)

**Review by iancooper — CHANGES_REQUESTED** [review]

**Comment by iancooper** [comment]
We have changes that will impact this one, as we are moving PostgreSql to the same model as other Outboxes. I am going to close this one as a result.

**Comment by SimonCropp** [comment]
@iancooper no worries. do you want more of these type of prs?

**Comment by iancooper** [comment]
Happy to keep taking PRs. This one just collided with a major restructuring of that code.

**Pull request closed without merging**